storager.go

```diff
@@ -77,6 +77,43 @@ func TestStorager(t *testing.T, store types.Storager) {
 			})
 		})
 
+		Convey("When Read a file with offset and size", func() {
```

Comment:
Maybe we also need tests for offset only and size only?

Reply:
Got it.